Fast Pair Locator Tag: update device model & add disablement instruction (#16182)
nrfconnect/sdk-nrf: samples/bluetooth/fast_pair/locator_tag/README.rst

      #. Start the ringing action on your device by tapping the :guilabel:`Play sound` button.
      #. Observe that **LED 1** is lit to confirm that the Android device is able to connect to your development kit after a clock synchronization.

> Disablement instructions

Reviewer: I'm not convinced about the "Disablement". Maybe it could be changes to something like: "disabling the locator tag" ?

Author: Sure, I can change it. Please note that this section is related to the following documentation from Google

https://support.google.com/android/answer/13658562?hl=en#zippy=%2Cdisable-the-tracker

You can also see the Apple AirTag disable instructions as an example

https://found.apple.com/airtag/disable

Reviewer: Thanks! "Disabling" retains the verb used in the articles you provided, so I would suggest changing as per suggestion

------------------------

To disable the locator tag device, complete the following steps:

.. note::
   Disablement instructions are used in Android and iOS to provide information on how to disable unknown trackers from each vendor.

>    These disablement instructions section is associated with the default debug device model registered for this sample (see the :ref:`fast_pair_locator_tag_google_device_model` section) and is used for demonstration purposes.

Reviewer 2: What you mean by `These disablement instructions section is associated with the default debug device model registered for this sample`?

> To disable the locator tag device, complete the following steps:
> 
> .. note::
>    Disablement instructions are used in Android and iOS to provide information on how to disable unknown trackers from each vendor.
>    These disablement instructions section is associated with the default debug device model registered for this sample (see the :ref:`fast_pair_locator_tag_google_device_model` section) and is used for demonstration purposes.

Reviewer: Is this a best place to include the note? Since above we're referring to listed steps (``complete the following steps:``), I feel those steps should appear right below the statement. Maybe we could rework the paragraph and get rid of the note completely? E.g.,

```suggestion
The following instructions explain how to disable unknown trackers from each vendor on Android and iOS devices.
They apply to the default debug device model registered for this sample (see the :ref:`fast_pair_locator_tag_google_device_model` section), and are used for demonstration purposes.
To disable the locator tag device, complete the following steps:

.. note::
```

Author: > The following instructions explain how to disable unknown trackers from each vendor on Android and iOS devices.

This sentence changed the meaning, our disablement instructions are only relevant for our model and the Nordic DK/Thingy:53. The previous sentence was supposed to be informational and explain why mobile platforms (iOS / Android) need disablement instructions.

Reviewer: thanks for clarifying - didn't catch that

> 1. Locate the Power switch on the device.

Reviewer: ```suggestion
1. Find the power switch on the device.
```

> #. Power off the locator tag by moving the switch to the off position.

Reviewer: ```suggestion
#. Turn off the locator tag by sliding the switch to the off position.
```

> #. Observe that on-device LEDs are turned off.

Reviewer: ```suggestion
#. Observe that the on-device LEDs are turned off.
```

      #. Start the ringing action on your device by tapping the :guilabel:`Play sound` button.
      #. Observe that **LED 1** is lit to confirm that the Android device is able to connect to your development kit after a clock synchronization.

Disabling the locator tag
-------------------------

> .. note::
>    Disablement instructions are used on mobile platforms like Android and iOS to provide information on how to disable unknown trackers from each vendor.

Reviewer: to be moved to the end of the section + as discussed offline
```suggestion
.. note::
   Vendor-specific instructions on disabling the unknown trackers are registered on mobile platforms like Android and iOS for each device model.
```

> The following disablement instructions apply to the default debug device model registered for this sample (see the :ref:`fast_pair_locator_tag_google_device_model` section), and are used for demonstration purposes.

Reviewer: ```suggestion
The following instructions on disabling the locator tag apply to the default debug device model registered for this sample (see the :ref:`fast_pair_locator_tag_google_device_model` section), and are used for demonstration purposes.
```
